config/jail.conf

```diff
@@ -958,6 +958,9 @@ port    = http,https
 logpath = %(syslog_authpriv)s
 backend = %(syslog_backend)s
 
+[routeros-auth]
+port = ssh
```

Comment:
Hmm... the banning port is ssh, but the message in your excerpt says "login failure from ... **via web**".
Doesn't it imply a http/https also? I mean a multi-port banning action will only ban the access from IP to port 22, so possible access to 80/443 are still allowed after ban (also if locally login takes place by ssh internally, but it is allowed due to local IP address, and it is good so).

Reply:
Ah you are right. I missed that. I will change it to `port = ssh,http,https`.